app/Event/EventJSONLDServiceProvider.php

```diff
@@ -68,7 +68,10 @@ public function register(): void
         $container->addShared(
             'event_jsonld_repository',
             function () use ($container): BroadcastingDocumentRepositoryDecorator {
-                $repository = new CacheDocumentRepository($container->get('event_jsonld_cache'));
+                $repository = new CacheDocumentRepository(
+                    $container->get('event_jsonld_cache'),
+                    $container->get('config')['cache']['milliseconds_between_retry'] ?? 0
```

Comment:
Maybe best to also make a PR with this key in `appconfig` & `udb3-vagrant`?

Reply:
For UDB3 Vagrant: https://github.com/cultuurnet/udb3-vagrant/pull/156
For appconfig: https://github.com/cultuurnet/appconfig/pull/324